lib/ProductOpener/Ingredients.pm

```diff
@@ -196,7 +196,7 @@ my %may_contain_regexps = (
 	lt => "sudėtyje gali būti",
 	lv => "var saturēt",
 	nl =>
-		"Dit product kan sporen van|bevat mogelijk sporen van|Kan sporen bevatten van|Kan sporen van|bevat mogelijk|sporen van",
+		"Dit product kan sporen van|bevat mogelijk sporen van|Kan sporen bevatten van|Kan sporen van|bevat mogelijk|sporen van|kan",
```

Comment:
Sorry for my late reply, do you remember which product had that? It would be useful to add a test for it.
cc our Dutch expert @aleene

Reply:
Using just one word "kan" is very open. The word might be used for other things. Would it be possible to check that first?

Reply:
The word "kan " (followed by a space) is not found in the taxonomies file. At least from this side, not false positives.